apps/music/js/ui/views/list_view.js

```diff
@@ -95,6 +95,8 @@ var ListView = {
       headerLi = document.createElement('li');
       headerLi.className = 'list-header';
       headerLi.textContent = this.lastFirstLetter || '?';
+      headerLi.setAttribute('role', 'heading');
+      headerLi.setAttribute('aria-level', '2');
```

Comment:
@eeejay Yes, just thought that's something related to the list elements but we can do it another bug.